app/src/InnovationWeek.php

```diff
@@ -1,26 +1,41 @@
 <?php
 
 use MaximeRainville\SilverstripeReact\ReactAdmin;
-use PharIo\Manifest\Requirement;
+use SilverStripe\Control\Controller;
 use SilverStripe\Core\Manifest\ModuleResourceLoader;
-use SilverStripe\View\Requirements;
 
 class InnovationWeek extends ReactAdmin
 {
     private static $url_segment = 'innovation';
 
     private static $menu_title = 'Innovation Week';
 
-    public function getProps(): array
-    {
-        return [
-            'mdUrl' => ModuleResourceLoader::singleton()->resolveURL('app/help/index.md')
-        ];
-    }
+    private static $url_handlers = [
+        // Only /admin/innovation/foo is a valid route
+        'foo//' => 'index',
+
+        // This route accepts parameters, so we give a wild card
+        'bar//$*' => 'index',
+    ];
 
-    public function getComponent(): string
+
+    public function getClientConfig()
     {
-        return 'InnovationWeek';
+        $baseLink = $this->Link();
+
+        return array_merge(parent::getClientConfig(), [
```

Comment:
The may asset-admin passes inital data to the front end is by using the LeftAndMain client config. This gets outputted as a JSON blob directly into the page. Not sure how I feel about this generally.